ui/components/MesheryPlayComponent.js

```diff
@@ -83,211 +83,184 @@ const styles = (theme) => ({
-          self.setState({ adapter: selectedAdapter.get(0) });
-          setAdapter({ selectedAdapter: selectedAdapter.get(0).name });
+          setAdapterState({ adapter: selectedAdapter[0] });
+          setAdapter({ selectedAdapter: selectedAdapter[0].name });
         }
       }
     };
   };
 
-  renderIndividualAdapter() {
-    const { meshAdapters } = this.props;
+  const renderIndividualAdapter = () => {
     let adapCount = 0;
     let adapter;
     meshAdapters.forEach((adap) => {
-      if (adap.adapter_location === this.props.adapter) {
+      if (adap.adapter_location === props.adapter) {
         adapter = adap;
         meshAdapters.forEach((ad) => {
           if (ad.name == adap.name) adapCount += 1;
         });
       }
     });
     if (adapter) {
-      const imageIcon = this.pickImage(adapter);
+      const imageIcon = pickImage(adapter);
       return (
-        <React.Fragment>
+        <>
           <MesheryAdapterPlayComponent
             adapter={adapter}
             adapCount={adapCount}
             adapter_icon={imageIcon}
           />
-        </React.Fragment>
+        </>
       );
     }
     return '';
-  }
-
-  render() {
-    const { classes, meshAdapters } = this.props;
-    let { adapter } = this.state;
-
-    if (meshAdapters.size === 0) {
-      return (
-        <NoSsr>
-          <React.Fragment>
-            <div
-              style={{
-                display: 'flex',
-                flexDirection: 'column',
-                alignItems: 'center',
-                justifyContent: 'center',
-                gap: '1rem',
-                height: '100%',
-                width: '100%',
-              }}
-            >
-              <Typography variant="h6">Adapter Unavailable</Typography>
-              <Typography variant="subtitle">Connect Meshery Adapter(s) in Settings</Typography>
-              <Button
-                variant="contained"
-                color="primary"
-                size="large"
-                onClick={this.handleConfigure}
-              >
-                <SettingsIcon className={classes.icon} />
-                Configure Settings
-              </Button>
-            </div>
-          </React.Fragment>
-        </NoSsr>
-      );
-    }
-
-    if (this.props.adapter && this.props.adapter !== '') {
-      const indContent = this.renderIndividualAdapter();
-      if (indContent !== '') {
-        return indContent;
-      } // else it will render all the available adapters
-    }
+  };
 
-    const self = this;
-    const imageIcon = self.pickImage(adapter);
+  if (meshAdapters.size === 0) {
     return (
       <NoSsr>
-        <React.Fragment>
-          <div className={classes.playRoot}>
-            <Grid container>
-              <Grid item xs={12} className={classes.paneSection}>
-                <TextField
-                  select
-                  id="adapter_id"
-                  name="adapter_name"
```

Comment:
Why we removed or made changes here?

Reply:
Hello @Yashsharma1911 We didn't changed anything here I guess the part is not showing here properly. You can check from line 210-226.
I just removed the <React.Fragment>